### PR TITLE
Add details popup window on each project

### PR DESCRIPTION
**Pull request by abdallahmalima**
In this part we implement the following functionality:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

- When the user clicks (or taps) the close (X) button, the popup disappears.

-  Use a JavaScript array to store all of the information for all projects.

- For each project store the following pieces of data in a JavaScript object: name, description, featured image, technologies, link to live version, and link to the source.

- Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
- Implement the popup window.
-

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    technologies:["CANOPY","Back","2015"],
+    languages:["html","css","javascript","github","ruby","bootstrap"],
+   },
+   {
+    name:"Uber Navigation",
+    description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    featured_image:"images/snapshort_project2.png",
+    link_to_live:"https://abdallahmalima.github.io/portfolio_mobile_first/",
+    link_to_source:"https://github.com/abdallahmalima/portfolio_mobile_first",
+    technologies:["CANOPY","Back","2015"],
```
Comment (tufoinnkuo10) [inline app.js:43]: - [ ] Great job. kindly ensure that the see live and see source buttons are well centralized in the mobile version

File: app.js [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    languages:["html","css","javascript","github","ruby","bootstrap"],
+   },
+   {
+    name:"Multi-Post Stories",
```
Comment (tufoinnkuo10) [inline app.js:47]: - [ ] please make your application responsive. switching from mobile to desktop we have a complete different version and vice versa. please ensure it has thesame version while switching from mobile like the original figma version. great job
see demo below
demo33

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @abdallahmalima 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @abdallahmalima 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**